stablehlo/reference/Element.h

```diff
@@ -41,8 +44,45 @@ class Element {
       : type_(type), value_(std::make_pair(value.real(), value.imag())) {}
 
   Element(const Element &other) = default;
+  Element() = default;
   /// @}
 
+  /// @}
+  /// \name Value Generators
+  /// @{
+
+  /// Get the specified value for the specified type.
+  template <typename V>
+  static Element getValue(Type type, V value) {
+    if (isSupportedSignedIntegerType(type)) {
+      return Element(
+          type, APInt(type.getIntOrFloatBitWidth(), value, /*isSigned=*/true));
+    } else if (isSupportedUnsignedIntegerType(type)) {
+      return Element(
+          type, APInt(type.getIntOrFloatBitWidth(), value, /*isSigned=*/false));
+    } else if (isSupportedFloatType(type)) {
+      APFloat floatVal((double)value);
+      bool roundingErr;
+      floatVal.convert(type.cast<FloatType>().getFloatSemantics(),
+                       APFloat::rmNearestTiesToEven, &roundingErr);
+      return Element(type, floatVal);
+    } else if (isSupportedComplexType(type)) {
```

Comment:
For complex type, use a `std::complex<V, V>` instead of `double`. This rhymes better with what an `Element` can internally store (`APInt, bool, APFloat, or std::pair<APFloat, APFloat>`).